core/src/main/java/bio/terra/pearl/core/service/survey/SurveyTaskDispatcher.java

```diff
@@ -189,8 +194,10 @@ public Optional<ParticipantTask> buildTaskIfApplicable(Enrollee enrollee,
         return Optional.empty();
     }
 
-    public static boolean isEligibleForSurvey(String eligibilityRule, EnrolleeRuleData enrolleeRuleData) {
-        return EnrolleeRuleEvaluator.evaluateRule(eligibilityRule, enrolleeRuleData);
+    public boolean isEligibleForSurvey(String eligibilityRule, EnrolleeRuleData enrolleeRuleData) {
```

Comment:
same comment as with consent forms -- we need to apply a default rule (that it only applies to subjects) if nothing is specified